torch/csrc/jit/mobile/import.cpp

```diff
@@ -424,17 +424,53 @@ void BytecodeDeserializer::parseMethods(
 
     static const c10::QualifiedName classPrefix = "__torch__.torch.classes";
```

Comment:
I'm a bit confused: if per your description the change is entirely FC, I assumed then that all the changes would take place in the serialization code.

However, if there's changes needed on import how can this feature be forward compatible?